packages/cloud-config/oem/08_boot_assessment.yaml

```diff
@@ -90,7 +89,7 @@ stages:
                   set extra_passive_cmdline="upgrade_failure"
                 else
                   set boot_assessment_tentative="yes"
-                  save_env -f "(${boot_assessment_blk})${boot_assessment}" boot_assessment_tentative
+                  save_env -f "${boot_assessment_file}" boot_assessment_tentative
```

Comment:
Damn it, this exact line is checked on tests, in fact the tests expect the boot_assessment_blk literal. I guess the literal the test checks could be changed to `set boot_assessment_tentative="yes"` or something similar